build/template-build-and-prep-automation.yaml

```diff
@@ -17,37 +17,14 @@ steps:
   inputs:
     version: 6.x
 
-#Workaround for Xamarin iOS/MAC build issue. See https://github.com/actions/runner-images/issues/6082
```

Comment:
Hosted agent has been updated to have this component by default. removing this made no difference in encountered errors